### PR TITLE
Show 1 decimal place in % complete if < 10%

### LINKED ISSUE #1250: Percent complete isn't advancing
Staging has 63 blocks mapped and is still at 0% complete. Is that too few to advance the count?

### PR DESCRIPTION
**Pull request by RickMohr**
Fixes #1250

### CODE REVIEW

File: src/nyc_trees/apps/home/views.py [hunk 1]
```diff
@@ -51,10 +51,11 @@ def _global_counts(past_week=False):
     else:
         blocks_mapped = Survey.objects.all().distinct('blockface_id').count()
     if blocks_mapped > 0:
-        blocks_percent = "{:.0%}".format(
+        fmt = "{:.0%}" if blocks_mapped >= 10 else "{:.1%}"
```
Comment (kdeloach) [inline src/nyc_trees/apps/home/views.py]: Should this check for percentage of blocks mapped instead of number of blocks mapped?

Comment (RickMohr) [inline src/nyc_trees/apps/home/views.py]: Thanks. Fixing...

Comment (RickMohr) [inline src/nyc_trees/apps/home/views.py]: Fixed.

File: src/nyc_trees/apps/home/views.py [hunk 2]
```diff
@@ -51,10 +51,11 @@ def _global_counts(past_week=False):
...
         blocks_mapped = Survey.objects.all().distinct('blockface_id').count()
     if blocks_mapped > 0:
-        blocks_percent = "{:.0%}".format(
-            float(blocks_mapped) / float(blocks_total))
+        fraction_mapped = float(blocks_mapped) / float(blocks_total)
+        fmt = "{:.0%}" if fraction_mapped >= .1 else "{:.1%}"
+        blocks_percent = fmt.format(fraction_mapped)
     else:
-        blocks_percent = "0%"
+        blocks_percent = "0.0%"
```
Comment (kdeloach) [inline src/nyc_trees/apps/home/views.py]: Minor, but I think we could safely remove this conditional to create a single code path.

**Comment by kdeloach** [comment]
+1

**Comment by RickMohr** [comment]
@azavea-bot rebuild

**Pull request merged**